benchmarks/gradOperator/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # SPDX-FileCopyrightText: 2023 NeoFOAM authors
 add_executable(gradOperator "gradOperator.cpp")
 
-set_target_properties(gradOperator PROPERTIES CXX_STANDARD 17)
+set_target_properties(gradOperator PROPERTIES CXX_STANDARD 20)
 
 target_compile_definitions(gradOperator PUBLIC WM_LABEL_SIZE=32 NoRepository WM_DP OPENFOAM=2306
                                                OMPI_SKIP_MPICXX)
```

Comment:
In OGL I use the following 
`target_compile_definitions(OGL PUBLIC WM_LABEL_SIZE=$ENV{WM_LABEL_SIZE}
                                      WM_$ENV{WM_PRECISION_OPTION})`
to get the label and scalar size from the environment. this could be a public NeoFOAMAdapter property. Than we don't have to set it explicetly everytime.

Reply:
thx